### DISCUSSION TITLE
Zimodem 3.7.1 - Wifi LED

### DISCUSSION THREAD

**Discussion by Biker1bob** [bozimmerman/Zimodem#91] [Q&A] [answered]
Hello all, Bo turned on discussions, I really thought a forum for this great firmware would be very helpful, and I would not have to pester Bo.. lol

I am using 3.7.1 - I have made #define SUPPORT_LED_PINS true  - So that I could get the WIFI indicator LED.  But I am finding that it is always on - if I am connected to wifi or not. 

"#ifdef SUPPORT_LED_PINS
" # ifdef GPIO_NUM_0
"#   define DEFAULT_PIN_AA GPIO_NUM_16
"#   define DEFAULT_PIN_HS GPIO_NUM_15
"#   define DEFAULT_PIN_WIFI GPIO_NUM_0
"# else
"#   define DEFAULT_PIN_AA 16
"#   define DEFAULT_PIN_HS 15
"#   define DEFAULT_PIN_WIFI 0
"# endif
"# define DEFAULT_HS_BAUD 38400
"# define DEFAULT_AA_ACTIVE LOW
"# define DEFAULT_AA_INACTIVE HIGH
"# define DEFAULT_HS_ACTIVE LOW
"# define DEFAULT_HS_INACTIVE HIGH
"# define DEFAULT_WIFI_ACTIVE LOW
"# define DEFAULT_WIFI_INACTIVE HIGH
"#endif

AA is what? AutoAnswer?   HS is HighSpeed.. now maybe my pin output needs to be reverse of Default? 

I have all wired up - I will post a picture of my wiring diagram tomorrow, need to adjust some errors I made.

**Comment by Biker1bob** [comment, marked as answer]
From what I see - there is no way to adjust the HIGH or LOW of the WIFI default in the AT commands? 

Also because I wanted to see if the light goes on or off.. There is no way to just DISCONNECT from the WIFI? You cannot Choose to be disconnected?

**Reply by bozimmerman** [reply]
You can disconnect WiFi by Failing to connect to WiFi.  If you are connected, you can enter ATW"no-ssid,no-pw", and it will immediately Disconnect, and then Fail to connect, leaving you in a disconnected state.

**Reply by Biker1bob** [reply]
Thanks Bo, I was able to do that and confirm it working. I am confident I have everything working well.
